HACKING.md

```diff
@@ -1,288 +1,169 @@
-# Goldstone Hacking Guide
+Goldstone Server Hacking Guide
+================
```

Comment:
This file has a Github Markdown extension, but this is using ReST format controls.  Confusing to anyone editing the source, and if it doesn't confuse the markdown processor now, it eventually will.